src/util/FactorUtil.js

```diff
@@ -155,8 +157,15 @@ function (Okta, TimeUtil) {
       return 'QUESTION';
     }
     if (provider === 'FIDO' && factorType === 'webauthn') {
-      return 'WINDOWS_HELLO';
-    }
+      if (this.settings != null && this.settings.get('features.webauthn') != null) {
+        webauthnEnabled = this.settings.get('features.webauthn');
+      }
+      if (webauthnEnabled) {
+        return 'U2F';
```

Comment:
Just curious - are Windows Hello and WebAuthn compatible? This may be an extreme edge case, but what happens if an org turns off the windows_hello flag and enables the new webauthn flag?

Reply:
We'll figure out whether to migrate existing windows hello factor enrollments to webauthn factor when we add Hello support for the new factor. As for the ff case, definitely a weird case, I'll follow up on it. (also tracked in https://oktainc.atlassian.net/browse/OKTA-186817).

Reply:
we should work on that jira soon, so we don't risk having an issue if both factors are enabled. Do we know when that jira will be worked on?

Reply:
yup, it's pending review